src/translations/installer_en.ts

```diff
@@ -235,6 +235,9 @@
     <message>
         <source>Lang_it</source>
         <translation>Italian</translation>
+    </message>
+            <source>Lang_pt</source>
+        <translation>Português</translation>
```

Comment:
```suggestion
        <translation>Portuguese</translation>
```
This is the English translation file, so this should be translated in English.